DP02_11_User_Packages/DP02_11_Working_with_user_packages.ipynb

```diff
@@ -7,9 +7,9 @@
    "source": [
```

Comment:
<p>The directory ~/WORK I think does exist for all users unless they deleted it, but ~/WORK/GitHub does not. Add a statement that they might have to create this directory first.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Done!</p>

Reply:
<p>Done</p>